airbyte-integrations/connectors/source-awtomic/main.py

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+import sys
+
+from airbyte_cdk.entrypoint import launch
+from source_awtomic import SourceAwtomic
+
+if __name__ == "__main__":
```

Comment:
`main.py` should invoke `run.py` as in `source-slack`. That's required for the connector to be published to pypi as a package and be available in `pyairbyte`.